src/bot.ts

```diff
@@ -16,7 +16,7 @@ const spotify = new Spotify({
 const lastfm = new Lastfm({
   key: process.env.LASTFM_API_KEY as string
 })
-const isLastFmUsing = process.env.LASTFM_API_KEY?.length && process.env.LASTFM_USERNAME?.length
+const isLastFmUsing = !!process.env.LASTFM_API_KEY?.length && !!process.env.LASTFM_USERNAME?.length
```

Comment:
:thinking: looks a bit off. what if we change it to explicit `Boolean` calls?
`Boolean(X) && Boolean(Y)` — like that since `process.env` values might be just empty strings and not `undefined`s